lib/identity/stateTransitions/identityCreateTransition/applyIdentityCreateTransitionFactory.js

```diff
@@ -20,8 +20,8 @@ function applyIdentityCreateTransitionFactory(
    * @return {Promise<void>}
    */
   async function applyIdentityCreateTransition(stateTransition) {
-    const output = stateTransition.getAssetLock().getOutput();
-    const outPoint = stateTransition.getAssetLock().getOutPoint();
+    const output = stateTransition.getAssetLock().getProof().getOutput();
```

Comment:
You need a special function to get output by Asset Lock Proof. Use state repository in case of Chain Asset Lock Proof